cmd/rook/main.go

```diff
@@ -232,6 +240,14 @@ func setLogLevel() {
 	capnslog.SetGlobalLogLevel(cfg.logLevel)
 }
 
+func logStartupInfo(cmdFlags *pflag.FlagSet) {
+	// log the version number, arguments, and all final flag values (environment variable overrides
+	// have already been taken into account)
+	flagValues := flags.GetFlagsAndValues(cmdFlags)
+	logger.Infof("starting rook %s with arguments '%s'", version.Version, strings.Join(os.Args, " "))
```

Comment:
seems like there is a place we print some arguments that wouldn't be needed anymore. Maybe that was just for standalone. Does this line still make sense? https://github.com/rook/rook/blob/master/pkg/clusterd/join.go#L34